src/sentry/mail/adapter.py

```diff
@@ -95,8 +96,20 @@ def get_sendable_user_objects(project):
         Return a collection of USERS that are eligible to receive
         notifications for the provided project.
         """
-        recipients_by_provider = NotificationSetting.objects.get_notification_recipients(project)
-        return recipients_by_provider.get(ExternalProviders.EMAIL, [])
+        user_ids = project.member_set.values_list("user", flat=True)
+        users = User.objects.filter(id__in=user_ids)
```

Comment:
So this is a little tricky. The most important thing is that `MailAdapter.get_sendable_user_objects` is contracted to return `User` ORM objects. Maybe that ought to change to `RpcUser` in the long run, but for now I want to leave `MailAdapter`'s interface alone as much as possible.

The reference to `User.objects`, which wasn't there before, is a side effect of swapping a call to `NotificationSetting.objects.get_notification_recipients` for `filter_to_accepting_recipients`. Previously, `get_notification_recipients` would have queried `User.objects` in the same way, but it no longer returns those `User` values.

Reply:
I'm ok with this -- ultimately we will have to change it, but given that it's not yet in our test coverage, and it will be (after the split db), it doesn't pose any risk other than something we will be mindful of.

This gives us the chance to get a meaningful simplification to actors committed, and then address this particular subsystem separately.